test/contracts/operator_lambdas.aes

```diff
@@ -0,0 +1,14 @@
+include "List.aes"
+
+contract C =
+    type state = int
+
+    function sum(l : list(bool)) : bool = List.foldl((||), false, l)
```

Comment:
sum? :p

Reply:
Will change this to `any`.

Reply:
Changed it to `any` here 642f5a4da32090f2e1dd277b9d9b84482b3b8942